src/components/Patient/LinkDepartmentsSheet.tsx

```diff
@@ -199,37 +227,38 @@ export default function LinkDepartmentsSheet({
                 {t("current_organizations")}
               </h3>
               <div className="space-y-2">
-                {currentOrganizations.map((org) => (
-                  <div
-                    key={org.id}
-                    className="flex items-center justify-between rounded-md border p-2"
-                  >
-                    <div className="flex items-center space-x-2">
-                      <Building className="h-4 w-4 text-blue-400" />
-                      <div className="flex flex-col">
-                        <span className="font-medium">{org.name}</span>
-                        {org.description && (
-                          <span className="text-xs text-gray-500">
-                            {org.description}
-                          </span>
-                        )}
+                {currentOrganizations[0] != null &&
+                  currentOrganizations.map((org) => (
+                    <div
+                      key={org.id}
+                      className="flex items-center justify-between rounded-md border p-2"
+                    >
+                      <div className="flex items-center space-x-2">
+                        <Building className="h-4 w-4 text-blue-400" />
+                        <div className="flex flex-col">
+                          <span className="font-medium">{org.name}</span>
+                          {org.description && (
+                            <span className="text-xs text-gray-500">
+                              {org.description}
+                            </span>
+                          )}
+                        </div>
                       </div>
+                      <Button
+                        variant="ghost"
+                        size="icon"
+                        onClick={() => removeOrganization(org.id)}
+                        disabled={isRemoving}
+                      >
+                        {isRemoving ? (
+                          <Loader2 className="h-4 w-4 animate-spin" />
+                        ) : (
+                          <Trash2 className="h-4 w-4 text-destructive" />
+                        )}
+                      </Button>
                     </div>
-                    <Button
-                      variant="ghost"
-                      size="icon"
-                      onClick={() => removeOrganization(org.id)}
-                      disabled={isRemoving}
-                    >
-                      {isRemoving ? (
-                        <Loader2 className="h-4 w-4 animate-spin" />
-                      ) : (
-                        <Trash2 className="h-4 w-4 text-destructive" />
-                      )}
-                    </Button>
-                  </div>
-                ))}
-                {currentOrganizations.length === 0 && (
+                  ))}
+                {currentOrganizations[0] == null && (
```

Comment:
Why the change here? BE response shouldn't be any different from other APIs, is it?